packages/react/src/components/combobox/combobox.tsx

```diff
@@ -87,51 +91,85 @@ const ArrowButton = styled(IconButton)<{ disabled?: boolean }>`
     height: var(--size-1x);
     padding: var(--spacing-half);
     position: absolute;
-    right: var(--spacing-half);
+    right: 0;
     width: var(--size-1x);
     
     &:hover {
         background-color: transparent;
     }
 `;
 
-type AutoCompleteMode = 'none' | 'inline' | 'list' | 'both';
+const ClearButton = styled(IconButton)<{ disabled?: boolean }>`
+    align-items: center;
+    background-color: transparent;
+    border: 0;
+    color: ${({ disabled, theme }) => (disabled ? theme.greys['mid-grey'] : theme.greys['dark-grey'])};
+    display: flex;
+    height: var(--size-1x);
+    padding: var(--spacing-half);
+    position: absolute;
+    right: calc(var(--size-1x) + var(--spacing-1halfx));
+    width: var(--size-1x);
+    
+    &:after {
```

Comment:
Le single `:` ici c'est du backward compatible, mais la syntaxe officielle des pseudo-elements c'est avec un double `::` (contrairement au pseudo-selectors qui utilise un seul). 

```suggestion
    &::after {
```